static/app/views/replays/detail/tabbedPanel.tsx

```diff
@@ -0,0 +1,96 @@
+import {Fragment, ReactNode, useState} from 'react';
+import styled from '@emotion/styled';
+
+import NavTabs from 'sentry/components/navTabs';
+import {
+  Panel as BasePanel,
+  Panel as BasePanelBody,
+  PanelHeader as BasePanelHeader,
+} from 'sentry/components/panels';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+type Props = {
+  tabs: {
+    name: string;
+    render: ReactNode;
+  }[];
+};
+
+function TabbedPanel({tabs}: Props) {
+  const [active, setActive] = useState<string>(tabs[0].name);
+  return (
+    <Panel>
+      <PanelTabs underlined>
+        {tabs.map(tab => (
+          <li
+            key={tab.name}
+            className={active === tab.name.toLowerCase() ? 'active' : ''}
+          >
+            <TabButton onClick={() => setActive(tab.name)}>
+              <PanelHeader
+                isActive={tab.name === active}
+                onClick={() => setActive(tab.name)}
+              >
+                {t(tab.name)}
+              </PanelHeader>
+            </TabButton>
```

Comment:
Should these be links? Similar to what we do in `<FocusTabs>` -- although now we would have 2 separate sets of tabbed panels.